cluster/addons/volume-group-snapshots/groupsnapshot.storage.k8s.io_volumegroupsnapshotclasses.yaml

```diff
@@ -0,0 +1,94 @@
+---
```

Comment:
It turns out we can't install VolumeGroupSnapshots CRDs in cluster addons because they are for cloud provider jobs. 
See the following.  Here's how we enabled VolumeSnapshot testing in a GCE cluster. However, GCE CSI driver does not support VolumeGroupSnapshot yet, so the job can't run there.
https://github.com/kubernetes/test-infra/blob/6b9389ee5a64a70556b31b4bf7d7557a831f1fc9/config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml#L106